### PR TITLE
Add SVG components for logos

### PR DESCRIPTION
**Pull request by Tijani-Dia**
This PR adds components for AreWeHeadlessYet and Slack's logos.

### CODE REVIEW

File: components/SVG/index.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import AreWeHeadlessYetLogo from './AreWeHeadlessYetLogo';
+import SlackLogo from './SlackLogo';
+
+export { AreWeHeadlessYetLogo, SlackLogo };
```
Comment (jhancock532) [inline components/SVG/index.ts:4]: This is nice, I will refactor my own SVG folder like this

**Review by jhancock532 — COMMENTED** [review]
Looks good to me, consider if you need added flexibility in the future with the SVG prop types

File: components/SVG/types.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+type SVGProps = {
+    color: string;
+};
+
+export default SVGProps;
```
[the remark this answers is not in the document]
Comment (Tijani-Dia) [reply components/SVG/types.ts:5]: Thanks, James, I've been inspired by your work in the careers site when building these components.

I noticed that specifying arbitrary width and height would change the image resolution, leading sometimes to a different output than expected. I only specify the viewBox attribute and set width and height in the parent element/container.

**Review by patrickcuagan — APPROVED** [review]
This looks good! Thanks Tidiane - just wanted to ask if these SVGs have been minimized?

If not, I use https://jakearchibald.github.io/svgomg/ but any minimizer will probably do

**Pull request closed without merging**